packages/lab/src/date-input/DateInput.css

```diff
@@ -225,6 +224,7 @@
   padding: 0;
   text-align: left;
   width: min-content;
```

Comment:
```suggestion
```

Reply:
Will this fix more problems? cc @mark-tate

Reply:
there are 2 width values in this style at present.